weave-js/src/components/Panel2/ChildPanelExportReport/utils.ts

```diff
@@ -82,3 +83,27 @@ export function formatUpdatedAt({
   // Handles when unit is "week", "month"
   return format(date, 'MMM dd');
 }
+
+/**
+ * Generates an empty config for a new slate report
+ * @param initialBlocks (optional) - initial contents for the report body
+ */
+export function getEmptyReportConfig(
```

Comment:
this is kind of a lazy copy of [this function](https://github.com/wandb/core/blob/a8121adf538f14b528dea86234898f10d4a7c22e/frontends/app/src/util/report/index.ts#L125) in core. If I brought over all the type definitions and variables the original method references it would be a bit unwieldy so I just copied over the bare necessities